packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -291,7 +291,8 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
     }
 
     if (element) {
-      this._setActiveElement(element);
+      // when we Set focus to a specific child, we should recalculate the alignment depend on its position
```

Comment:
Actually, I think this makes sense. When focus is explicitly set via the imperative `focusElement` call, reset the alignment.

But need to confirm this code path isn't called when up/down arrow handlers are executed.

Reply:
Nope, just looked at the code. `_moveFocus` calls `focusElement`.

Reply:
This causes a change in behavior for the scenario mentioned above. The "can use arrows bidirectionally" test fails

Reply:
thanks for your comments
I created new function in the focuszone which control the alignment state to decide if we want to keep the previous alignment or we need to update it depends on the position of the new active element